### PR TITLE
Resprites intercoms

### PR DESCRIPTION
**Pull request by Quantum-M**
## About The Pull Request

Re-srpites the intercoms (both regular and wideband)
## Why It's Good For The Game
Hopefully more nicer and distinct sprites.

Main Intercoms:
image
Wall frame/unbuilt versions:
image

## Changelog
:cl: Quantum-M, Retlaw34
tweak: Resprites intercoms
/:cl:

### CODE REVIEW

**Review by triplezeta — APPROVED** [review]

**Review by triplezeta — CHANGES_REQUESTED** [review]
actually no, i rescind that, there's some things i didnt pick up earlier

**Review by Vexylius — CHANGES_REQUESTED** [review]
"Wideband intercoms still need to be changed so that they look normal on walls again". After that it can be merged. Pulling it out of testmerge machine.

**Review by Fikou — CHANGES_REQUESTED** [review]
add the table version as a subtype or automatically change the icon depending on what surface it was made on

**Comment by Vexylius** [comment]
I would merge it. This can be added as a later feature

File: whitesands/code/game/objects/items/devices/radio/intercom.dm [hunk 1]
```diff
@@ -21,4 +21,5 @@
 /obj/item/wallframe/intercom/wideband
 	name = "wideband relay frame"
 	desc = "A detached wideband relay. Attach to a wall and screw it in to use."
+	icon_state = "intercom-wideband"
 	result_path = /obj/item/radio/intercom/wideband/unscrewed
```
Comment (Fikou) [inline whitesands/code/game/objects/items/devices/radio/intercom.dm]:
```suggestion
	result_path = /obj/item/radio/intercom/wideband/unscrewed

/obj/item/wallframe/intercom/wideband/table
	icon_state = "intercom-wideband-table"
```

**Review by Vexylius — APPROVED** [review]
😎👍

**Review by triplezeta — APPROVED** [review]
these look good enough ingame

**Review by Fikou — APPROVED** [review]

**Pull request merged**